eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -31,8 +31,17 @@ steps:
         displayName: Set up ILC emulation environment
 
     - ${{ if eq(parameters.archType, 'wasm') }}:
-      - script: $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) ${{ parameters.osGroup }} $(crossArg) $(_officialBuildParameter) ci tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }}
-        displayName: Build runtime tests
+      - ${{ if eq(parameters.platform, 'browser_multithread_wasm_win') }}:
```

Comment:
It would be good to avoid this duplication. We can add a parameter to these post-build steps like `wasmEnableThreadsArg: /p:WasmEnableThreads=true` and use it here (and below to excluded the libs tests).

Reply:
done